docs/concepts.rst

```diff
@@ -202,8 +202,8 @@ Caching and Protocols
 Whenever Bionic computes an entity's value, it automatically caches that value
 in memory (in case you access it again in from the same ``Flow`` object) and in
 persistent storage (in case you want to access it later, perhaps after
-restarting your script or notebook).  Currently the only supported location for
-persistent storage is your computer's hard disk.
+restarting your script or notebook).  Currently the only supported locations for
+persistent storage are your computer's hard disk and gcs.
```

Comment:
nit: s/gcs/Google Cloud Storage/

Also, maybe we no longer need to phrase this in terms of "only".